docs/root/operations/performance.rst

```diff
@@ -70,3 +70,20 @@ The watchdog emits statistics in both the *server.* and *server.<thread_name>.*
 
   watchdog_miss, Counter, Number of standard misses
   watchdog_mega_miss, Counter, Number of mega misses
+
+
+Heap Shrink
+-----------
+
+Envoy by default use tcmalloc as memory allocator which provides function to return unused pages to the
+OS and reduce the physical memory usage. Currently envoy could shrink heap when xDS response
+is received and also when an admin handler is hit. 
```

Comment:
what admin handler? Ref link please?

Reply:
Updated: add ref to admin interface.
Also to your question: shrink is called on any handler, including but not limited to config_dump